inc/hero-text-widget.php

```diff
@@ -127,7 +127,7 @@ public function form( $instance ) {
 		<script>
 			( function ( $ ) {
 
-				// This let us know that we appended a new widget to reset sortables.
+				// Let our script know a widget has been added to make its URL search input work.
```

Comment:
@jonathanbardo I updated the comment here to better clarify why the inline script exists.